src/js/editor/editor.js

```diff
@@ -640,8 +642,27 @@ class Editor {
     return false;
   }
 
+  handleCopy(event) {
+    const { clipboardData } = event;
+    const range = this.cursor.offsets;
+    const mobiledoc = this.post.cloneRange(range);
+    const html = `<div data-mobiledoc='${JSON.stringify(mobiledoc)}'></div>`;
+    clipboardData.setData('text/html', html);
+    event.preventDefault();
```

Comment:
Yeah.
The preventDefault is necessary for the text/html that we set on the above line to be preserved.
Rendering the HTML with the HTML Renderer and inserting it in the div above would preserve copy-to-outside-world functionality.